common/src/main/java/ac/grim/grimac/utils/nmsutil/JumpPower.java

```diff
@@ -14,26 +14,28 @@
 
 @UtilityClass
 public class JumpPower {
-    public static void jumpFromGround(@NotNull GrimPlayer player, @NotNull Vector3dm vector) {
+    public static void jumpFromGround(@NotNull GrimPlayer player, @NotNull double[] vectorArr) {
```

Comment:
`@NotNull double[]` means an array of `@NotNull double`
```suggestion
    public static void jumpFromGround(@NotNull GrimPlayer player, double @NotNull [] vectorArr) {
```